packages/explat-client/src/test/index.ts

```diff
@@ -0,0 +1,47 @@
+/**
+ * External dependencies
+ */
+// This is required to fix the "regeneratorRuntime is not defined" error
+import '@automattic/calypso-polyfills';
+
+/**
+ * Internal dependencies
+ */
+import {
+	createExPlatClient as createBrowserExPlatClient,
+	createSsrSafeDummyExPlatClient,
+} from '../create-explat-client';
+
+beforeEach( () => {
+	jest.resetModules();
+} );
+
+function setBrowserContext() {
+	// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+	// @ts-ignore
+	global.window = {};
+}
+
+function setSsrContext() {
+	// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+	// @ts-ignore
+	global.window = undefined;
+}
```

Comment:
It'd be good to put these functions in the common utility file rather than duplicate them.